test/val/val_image_test.cpp

```diff
@@ -2262,6 +2262,8 @@ TEST_F(ValidateImage, ReadNeedCapabilityImageCubeArray) {
       "ImageRead"));
 }
 
+#if 0
+// TODO(atgoo@github.com) Disabled until the spec is clarified.
 TEST_F(ValidateImage, ReadWrongResultType) {
```

Comment:
If you have a broken test that you cannot fix right away, you can add the DISABLED_ prefix to its name. 

https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#temporarily-disabling-tests

But this is fine for now.